Tasks/Gradle/CodeAnalysis/SonarQube/parameter-helper.ts

```diff
@@ -0,0 +1,93 @@
+/// <reference path="../../../../definitions/vsts-task-lib.d.ts" />
+
+import tl = require('vsts-task-lib/task');
+
+import {ToolRunner} from 'vsts-task-lib/toolrunner';
+
+import {SonarQubeEndpoint} from './endpoint';
+import sqCommon = require('./common');
+
+export class SonarQubeParameterHelper {
```

Comment:
The file name convention is ussually ParameterHelper (or lowercase if you insist :)).